extension/test/unit/make-payment.handler.spec.js

```diff
@@ -0,0 +1,208 @@
+      encryptedSecurityCode: 'adyenjs_0_1_25$d/OMm3cbLOV4Dk9+a7rmJjZBaOsDCka6bTiuDG/JeeBuKYNjocNRSFnlPk2oV2+zUZWCHVpiFNx2ws9hDiNz9uxllO3TUpRk3whZpj30a7WUjXYOEXYgynXVtqpULXB5zP8Ro+1jP1Dr2Zbt00N8WPfehz/BnfsOu4fn2DsMhrIDlldYucsB50q1HzyLCa5IUFIsPaGQFV6TwzInLbhvEt4Nq7P1WZ/KaqJhwYURNgU89cjLUtboqIw/NGoDxG8XzdfCyGQE4yJYItgq71kpzdpIl0onxoaOaBBOaAo6ig3zNVDXl667LYiCdO049zIysS0eqJrTfzyw3qQDbiz1OQ==$5lxjuK68dSuG9jXL+sGufH2X1m2P8Z+MpswXgz28Z9OMuXIqUUFStx0EGA5WFCCZyEjknIdMd6FzbUTSji2mVYbbZWLvcxuHb/xsERQeltO1W7joM0co6GHZcVg+Vr8wp4foUCthX1y5zcIcLDVw4L73CZeRjBZyyGGTeOFV4zAr0UAYKhjQbWHQ7d/SyZuRSZGk2NL6u/eY8bf5ogSHnkYSRtVJYd95WCBhpiZR1AWrjpZBJtDqh2q53zHZuWtwmIJiGO7lZZdDdc/TpbOsLpJWcOXq6UYPiwyfIuE8U6xKyLTumkHNOBv8vDHT2PRlyvufUcrp7XthYBpHJqADjD0jvb8Kn80tL7/RDmbCtTUzHq1GFlzB25nF4JErDPgUZ+BiSI7SlMGmeJdoy9RKP2DIKZEEiw1/w3dA/4ea5D7hZ3D53LKBJTEO7uyqTJI+/M1NuAKSeeFLZE9X95+AFpx3RDGKCG380PAH'
+    },
+    browserInfo: {
+      acceptHeader: '*/*',
+      colorDepth: 24,
+      language: 'en-US',
+      javaEnabled: false,
+      screenHeight: 1050,
+      screenWidth: 1680,
+      userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.92 Safari/537.36',
+      timeZoneOffset: -120
+    },
+    amount: {
+      currency: 'EUR',
+      value: 1000
+    }
+  }
+  /* eslint-enable max-len */
+
+  beforeEach(() => {
+    scope = nock(`${config.adyen.apiBaseUrl}`)
+  })
+
+  it('when resultCode from Adyen is "Authorized", '
+    + 'execute() should return actions create transaction, add interface interaction and set custom field',
+    async () => {
+      scope.post('/payments')
+        .reply(200, paymentSuccessResponse)
+
+      const ctpPaymentClone = _.cloneDeep(ctpPayment)
+      ctpPaymentClone.custom.fields.makePaymentRequest = JSON.stringify(makePaymentRequest)
+
+      const response = await execute(ctpPaymentClone)
+
+      expect(response.actions).to.have.lengthOf(3)
+      const addInterfaceInteraction = response.actions.find(a => a.action === 'addInterfaceInteraction')
+      expect(addInterfaceInteraction.fields.type).to.equal('makePayment')
+      expect(addInterfaceInteraction.fields.request).to.be.a('string')
+      expect(addInterfaceInteraction.fields.response).to.be.a('string')
+      expect(addInterfaceInteraction.fields.createdAt).to.be.a('string')
+      const request = JSON.parse(addInterfaceInteraction.fields.request)
```

Comment:
could you please add spaces to separate interaction, request, and transaction expect statements because of the readability.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/257/commits/d20ca52bf768e5b5b9d96ba20e252fa12f55f8bf